docs/providers.md

```diff
@@ -9,31 +9,33 @@ Currently, we have readers to accept metadata in the following formats:
 * XML
 * CSV
 * MARC21 binary
+* JSON (not tested)
 
-We have data mappings in existence or underway for MODS, MARCXML/MARC21, TEI, and FGDC.
+We have data mappings in existence for MODS, MARC21, TEI, FGDC, IIIF, and a number of local schema in CSV.
 
-## Record requirements 
+## DLME Object Record requirements
 
 A "minimum viable record" for DLME currently requires the following:
 
-* Title (or brief description)
-* Identifier (for the item or record)
-* Provider (the entity providing DLME with the record, including aggregators)
-* Data Provider (the entity that holds the item)
+* `Title` (or brief description)
+* `Identifier` (for the item or record)
+* `Provider` (the entity providing DLME with the record, including aggregators)
+* `Data Provider` (the entity that holds the item)
 
 We strongly recommend the following in addition to the minimum requirements above:
 
-* Is Shown At (a link to a digital object or an in-depth description of the item)
-* Preview (an image thumbnail depicting the item)
+* `Is Shown At` (a link to a digital object or an in-depth description of the item)
+* `Preview` (a link to an image thumbnail depicting the item)
 
-If certain information such as identifiers or links to digital objects or image thumbnails is unavailable from your metadata itself, we will likely ask for additional information about how to gather this data to build out quality records.
+If certain information such as identifiers, links to digital objects, or image thumbnails are unavailable from your metadata, we will likely ask for additional information about how to gather this data to build out quality records (e.g. with this identifier, and this URL structure, you can retrieve a thumbnail URL).
 
-More in-depth documentation is available below:
+## In-Depth Data Provider Documentation
 
-* [Application profile](application_profile.md)
-* [Metadata mappings](https://docs.google.com/spreadsheets/d/1Sp7uMHizVX7xN7xN9mm-vgEuESQBovXO-qenAo_TV-w/edit)
+* [Metadata ingested into the DLME needs to be mapped to the DLME Metadata Application Profile (MAP), or the available classes, fields, and definitions (including requirements) for objects described in DLME. You can read more details on the DLME MAP at this link](application_profile.md).
```

Comment:
Can we link just part of these sentences? This fields a little unwieldly.

Reply:
Sure